app/services/idv/agent.rb

```diff
@@ -8,7 +8,8 @@ def proof_resolution(
       document_capture_session,
       should_proof_state_id:,
       trace_id:,
-      document_expired:
+      document_expired:,
+      flow_path: 'standard'
```

Comment:
Same note as in https://github.com/18F/identity-idp/pull/5690#discussion_r767721184, though I expect (hope) that the tests will fail on the mismatch of keyword arguments being passed to `ResolutionProofingJob`.